packages/tocco-ui/src/SomeOfButton/SomeOfButton.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import _round from 'lodash/round'
+
+import Button from '../Button'
+import {
+  inkPropTypes,
+  stylingLook
+} from '../utilStyles'
+
+const affixSystemUnit = value => {
+  const log10 = Math.log10(value)
```

Comment:
Isnt it possible to create an object with the units and loop over it? This would increase readability.
im thinking about something like this:

`
[{value: 3, symbol: 'k'}, {value: 9, symbol: 'M'}]

`

Reply:
Thanks for inspiration.